app.py

```diff
@@ -1,21 +1,57 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+import requests
+import json
+import Trello
+import os
 
 app = Flask(__name__)
-app.config.from_object('flask_config.Config')
 
-@app.route('/', methods = ['POST', 'GET', 'PUT', 'DELETE'])
+@app.route('/', methods=['GET', 'POST', 'PUT'])
```

Comment:
Is there any reason to expect POST or PUT calls to this endpoint?

Reply:
None whatsoever. Removed